GoogleTimeline/ImportGoogleTimeline.js

```diff
@@ -8,6 +8,7 @@ import {LocationData} from '../utils/LocationData';
 class ImportGoogleTimeline extends Component {
   importFinished = (reason, notFailure) => {
     let title, description;
+    reason = reason.toString();
```

Comment:
js passes by value, it won't change `reason`, but `reason` is string already?

Reply:
I changed all callers to pass a fixed string as internal error messages are not useful.
This just prevents future errors as it ensure reason will always be a string.